components/button/demo/floating-buttons.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../demo/demo-page.js';
+			import '../button.js';
+			import '../floating-buttons.js';
+		</script>
+	</head>
+	<body unresolved>
+		<d2l-demo-page title="d2l-floating-buttons">
+			<h2>Floating Buttons</h2>
+
+			<d2l-button id="btn-remove-floating">Remove Floating Buttons</d2l-button>
+			<d2l-button id="btn-append-floating">Append Floating Buttons</d2l-button>
+			<d2l-button id="btn-append-new">Append New Button</d2l-button>
+			<d2l-button id="btn-remove-last">Remove Last Button</d2l-button>
+			<d2l-button id="btn-insert-content-above">Insert Content</d2l-button>
+			<d2l-button id="btn-remove-content-above">Remove Content</d2l-button>
+
+			<d2l-demo-snippet>
+				<div class="content">
+					<p>I love beer!</p>
+				</div>
+				<d2l-floating-buttons id="wc-buttons">
+					<d2l-button id="btn-make-more">Make more Beer!</d2l-button>
+				</d2l-floating-buttons>
+			</d2l-demo-snippet>
+
+		</d2l-demo-page>
+		<script type="module">
+			let page_buttonCount = 0;
+			let page_floatingButtons;
+			function appendButton() {
+				const floatingButtons = document.getElementById('wc-buttons');
+				if (floatingButtons) {
+					page_buttonCount += 1;
+					const newButton = document.createElement('button');
+					newButton.textContent = `Beer ${page_buttonCount}`;
+					floatingButtons.appendChild(newButton);
+				}
+			}
+			window.requestAnimationFrame(() => {
```

Comment:
Is this `requestAnimationFrame` still needed?  I suspect it should be possible to wire up the click handles right away now.